std/math.d

```diff
@@ -1068,16 +1074,16 @@ private T tanImpl(T)(T x) @safe pure nothrow @nogc
  *      $(TR $(TD $(NAN))    $(TD $(NAN))  $(TD yes))
  *  )
  */
-real acos(real x) @safe pure nothrow @nogc
+real acos(real radians) @safe pure nothrow @nogc
```

Comment:
`radians` doesn't make sense for the inverse functions (it applies to the return value).

Reply:
good catch that was a silly mistake